packages/web-container/index.html.ejs

```diff
@@ -7,9 +7,10 @@
   <title><%= data.title %></title>
   <link rel="manifest" href="manifest.json" crossorigin="use-credentials">
   <% Object.keys(data.bundle.css).forEach((s) => { %>
-    <link href="<%- data.bundle.css[s] %>" rel="stylesheet">
+    <link href="<%- data.bundle.css[s] %>?<%= data.compilationTimestamp %>" rel="stylesheet">
```

Comment:
CDN is always problematic because it's a potential attack vector.

Reply:
Yes, but is it a recommendation/general industry practice to never use these cdn's? (I'm a n00b, so I'm curious to know)

Reply:
some comapanies do not allow to call external scripts, this applies if we use a CDN, at a later point we could add a option to the build process to define the url

Reply:
I can add this as an option in the build process.